lib/update/system/system.go

```diff
@@ -713,3 +804,8 @@ func unpack(packages update.LocalPackageService, loc loc.Locator) error {
 	}
 	return trace.Wrap(pack.Unpack(packages, loc, path, nil))
 }
+
+// tctlScript is the template of the script that invokes tctl binary with
+// configuration supplied via an environment variable.
+var tctlScript = template.Must(template.New("tctl").Parse(`#!/bin/bash
+TELEPORT_CONFIG={{.conf}} {{.path}} "$@"`))
```

Comment:
Will `path` be properly escaped if it contains special characters? Ditto for teleport configuration.

Reply:
I don't think path can contain special characters. It points to the unpacked teleport package directory inside our state dir, and if somebody overrides state dir to have special characters in it, I think they'll have bigger problems than non-working tctl script.

And the configuration is base64-encoded so it doesn't need to be escaped.